.husky/pre-commit

```diff
@@ -1,7 +1,6 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-yarn lint-staged
-yarn lint
-yarn test
+yarn lint-staged --verbose
 yarn build
+git add dist
```

Comment:
Doesn't this cause warnings?

Reply:
Yea it needs some more work. I spent a lot of time making it work smoothly across all platforms before we upgraded.

I'm a bit disappointed that I (and others) weren't able to easily contribute anymore. Having to also deal with dev life cycle stuff makes contributing very hard.

Reply:
replaced `husky` and `lint-staged` altogether, using `lefthook` instead.